### ISSUE TITLE
Make it easy to plot paths using e.g. WCSAxes

### ISSUE THREAD

**Issue by astrofrog** [radio-astro-tools/pvextractor#60]
Need to show this in the documentation. This should work for pixel and skycoord paths. We can make use of `to_patches()`.

**Comment by keflavich** [comment]
Example in real life use:
https://github.com/keflavich/APEX_CMZ_H2CO/blob/master/analysis/orbit_pv.py#L292

**Comment by astrofrog** [comment]
Cool - it would be nice to have a convenience function for a quick view (that could use WCSAxes internally)

**Comment by keflavich** [comment]
:+1: 

A method on the `Path`?:
```
def show_on_axis(ax, separation, **kwargs):
        patches = P.to_patches(separation, wcs=self.wcs, clip_on=True, **kwargs)
        patchcoll = matplotlib.collections.PatchCollection(patches, match_original=True)

        c = ax.add_collection(patchcoll)
        return c
```
Customizing the patch properties is nontrivial, though.  I'm also not 100% sure I've gotten the properties right - I think `Path`s have WCS attributes, but I don't know whether `clip_on` is necessary.

**Comment by keflavich** [comment]
This tutorial shows some examples:
https://github.com/radio-astro-tools/tutorials/blob/02d56e89151f620d328851fd128d94267b979c58/PVDiagramPlotting.ipynb

**Comment by keflavich** [comment]
Solved in #102
